### DISCUSSION TITLE
Tag new faces easily

### DISCUSSION THREAD

**Discussion by naub1n** [photoprism/photoprism#3612] [General]
Hi,

Lot of faces are not in face cluster.
In Photoprism, it's not possible to tag manually multiple faces.

I go to "People" >> "New" >> "Show all new faces" but Photoprism show all photos with new faces, not only faces like in "People" tab in "Edit" section of single photo.

I would like to select multiple faces and tag them all once.

Can you help me ?

Thank you very much for your great job!

**Comment by graciousgrey** [comment]
Hey :)

Many improvements are planned for facial recognition:

- https://github.com/photoprism/photoprism/issues/1548
- https://github.com/photoprism/photoprism/issues/1601
- https://github.com/photoprism/photoprism/issues/1679

Feel free to leave a :+1: on the issues you are interested in.

**Reply by naub1n** [reply]
ahh thx !!!
I have searched in issues and roadmap before posting new issue but apparently not enough !! Sorry !! 
#1679 them to be exactly what I want.
